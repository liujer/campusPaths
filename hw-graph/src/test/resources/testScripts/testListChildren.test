# Tests the method listChildren of DLGraph by adding nodes with
# children and checking if the children are outputted correctly
CreateGraph A
AddNode A one
AddNode A two
AddNode A three
AddEdge A one two e12
AddEdge A one three e13
ListChildren A one
ListChildren A two
ListChildren A three
CreateGraph B
AddNode B one
AddNode B two
AddNode B three
AddEdge B one two e12
AddEdge B three two e32
ListChildren B one
ListChildren B two
ListChildren B three
CreateGraph C
ListChildren C one