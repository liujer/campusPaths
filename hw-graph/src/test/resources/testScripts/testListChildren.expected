# Tests the method listChildren of DLGraph by adding nodes with
# children and checking if the children are outputted correctly
Created graph A
Added node one to A
Added node two to A
Added node three to A
Added edge e12 to A
Added node e13 to A
ListChildren of one in A output: three two
ListChildren of two in A output:
ListChildren of three in A output:
Created graph B
Added node one to B
Added node two to B
Added node three to B
Added edge e12 to B
Added edge e32 to B
ListChildren of one in B output: two
ListChildren of two in B output:
ListChildren of three in B output: two
Created graph C
one does not exist in graph C